### ISSUE TITLE
Icon resets on macOS with every update

### ISSUE THREAD

**Issue by calebmeyer** [dhanishgajjar/vscode-icons#83]
I love these icons, but when I set an icon for vs code, it resets with every update. Is there any way to keep the icon change across updates of VS Code?

**Comment by esteban-url** [comment]
try quitting the app completely, then change the icon.

that just worked for me. good luck!

**Comment by nesimtunc** [comment]
Actually it resets even when the VSCode syncs the settings across the devices. So, I stopped to use it.

**Comment by calebmeyer** [comment]
> try quitting the app completely, then change the icon.
> 
> that just worked for me. good luck!

Yeah, I can always change it (and that's what I've been doing) by closing VS Code and re-applying the proper icon. But I'd like to know if there's a way to make it more permanent, or if updates are going to break it every time.
